pkg/R/column.R

```diff
@@ -0,0 +1,108 @@
+#' Column Class
+
+#' @include jobj.R
+NULL
+
+setOldClass("jobj")
+
+#' @title S4 class that represents a DataFrame column
+
+#' @rdname column-class
+#'
+#' @param jc reference to JVM DataFrame column
+#' @export
+setClass("Column",
+         slots = list(jc = "jobj"))
+
+setMethod("initialize", "Column", function(.Object, jc) {
+  .Object@jc <- jc
+  .Object
+})
+
+column <- function(jc) {
+  new("Column", jc)
+}
+
+# TODO: change Dsl to functions once update spark-sql
+col <- function(x) {
+  column(callJStatic("org.apache.spark.sql.Dsl", "col", x))
+}
+
+alias <- function(col, name) {
```

Comment:
alias, cast and also approxCountDistinct, countDistinct should become S4 functions